src/kubernetes.h

```diff
@@ -28,16 +28,21 @@
 #include "configuration.h"
 #include "environment.h"
 #include "json.h"
+#include "reader.h"
 #include "updater.h"
 
 namespace google {
 
-class KubernetesReader {
+class KubernetesReader : public MetadataReader {
  public:
   KubernetesReader(const MetadataAgentConfiguration& config);
   // A Kubernetes metadata query function.
   std::vector<MetadataUpdater::ResourceMetadata> MetadataQuery() const;
 
+  // Validates that the reader is configured properly.
```

Comment:
How about just:
```cpp
// Validates the relevant configuration and returns true if it's correct.
```
? Also in other files.

Reply:
Updated.